### PR TITLE
Create Note UI

### LINKED ISSUE #3: Create new note UI

### PR DESCRIPTION
**Pull request by kamolesh97**
closes #3 

@yedhink _a please review.

#### demo:

https://user-images.githubusercontent.com/71279916/217310787-dc008c0d-9e11-4beb-9b66-b578702be3ce.mov

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 1]
```diff
@@ -1,48 +1,63 @@
...
+
+  const handleSubmit = async values => {
+    try {
+      setNotes(notes => [...notes, values]);
+      Toastr.success(t("notes.noteCreateSuccess"));
+      onClose();
+    } catch (err) {
+      logger.error(err);
+    }
+  };
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:21]: Why's this an `async` block? What `async` operation is being performed here?

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 2]
```diff
@@ -1,48 +1,63 @@
...
+          <Pane.Footer>
+            <Button
+              className="mr-3"
+              disabled={isSubmitting}
+              label="Save changes"
+              loading={isSubmitting}
+              style="primary"
+              type="submit"
+            />
+            <Button label="Cancel" style="text" onClick={onClose} />
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:55]: https://academy.bigbinary.com/learn-react/react-best-practices/formik-forms#formik-cancel-button-type

**Review by yedhink — COMMENTED** [review]
@kamolesh97

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx (hunk 1 above)
Comment (kamolesh97) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:21]: @yedhink no async operation here, just the code from wheel and forgot to remove it in this PR, have removed in the upcoming PR.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:21]: Ok. Got it. Thanks for clarifying the same.

**Pull request merged**